src/views/RecIM/components/List/Listing/index.js

```diff
@@ -77,60 +83,100 @@ const SeriesListing = ({ series }) => {
 };
 
 const ActivityListing = ({ activity }) => {
-  let registrationStart = DateTime.fromISO(activity.RegistrationStart);
+  const [activityType, setActivityType] = useState();
+  const [currentCapacity, setCurrentCapacity] = useState(
+    <span style={{ display: 'inline-block' }}>
+      <GordonLoader size={15} />
+    </span>,
+  );
+  useEffect(() => {
+    const loadActivityType = async () => {
+      let activityTypes = await getActivityTypes();
+      setActivityType(
+        activityTypes.find((activityType) => activityType.ID === activity.TypeID).Description,
+      );
+    };
+    const calculateCurrentCapacity = async () => {
+      let fullActivity = await getActivityByID(activity.ID);
```

Comment:
Should rework the route to include this information on initial page load. Our eager loading philosophy is kinda broken here, but that is ok as the loading is not too bad right now.